components/brave_webtorrent/extension/brave_webtorrent.tsx

```diff
@@ -16,6 +16,15 @@ import App from './components/app'
 // Constants
 import { TorrentsState } from './constants/webtorrentState'
 
+// This is a hack that's needed for lazy loading
+// Basically we need the browser to restart the navigation, so we redirect first
+// to brave_webtorrent2.html and have that rewrite the URL to
+// brave_webtorrent.html
```

Comment:
The reason for this is the URL rewrite code gets hit before everything including the network throttle.  We have a similar hack for ethereum-remote-client.